### PR TITLE
[Container] Small typos

### PR DESCRIPTION
**Pull request by chadyred**
https://symfony.com/doc/current/reference/dic_tags.html#kernel-event-subscriber

### CODE REVIEW

**Review by OskarStark — APPROVED** [review]

File: service_container.rst [hunk 1]
```diff
@@ -1257,7 +1257,7 @@ unique string as the key of each service config:
         event_subscribers:
             namespace: App\Domain\
             resource: '../src/Domain/*/EventSubscriber'
-            tags: [event_subscriber]
+            tags: [kernel.event_subscriber]
```
Comment (xabbuh) [inline service_container.rst]: I think that the example deliberately used `event_subscriber` as a tag (matching the custom `command_handler` tag used above).

**Review by xabbuh — CHANGES_REQUESTED** [review]

File: service_container.rst (hunk 1 above)
Comment (chadyred) [reply service_container.rst]: Yeah, I checkout this change, thanks

**Review by xabbuh — APPROVED** [review]

**Comment by xabbuh** [comment]
Good catch, thanks @chadyred.

**Comment by chadyred** [comment]
You are welcome @xabbuh, thanks for your time and if you have a little bit more time, my old PR in the same spirit :horse: https://github.com/symfony/symfony-docs/pull/19395 :D

**Pull request merged**